app/views/user_mailer/password_reset.html.erb

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Apologies - I just noticed this is an email template. It only needs the content inside the body tag too but does not use the application layout.